### PR TITLE
alc opts on master

### PR DESCRIPTION
**Pull request by baiti**
same procedure just on master as opposed to release. Here it works as expected. Perhaps master is anyway the right branch for this function. If yes, review it and merge it and then let's throw away the one that was based on release, however, it would certainly be of help if I understood what I did wrong and what I should have done differently.

### CODE REVIEW

File: main/src/cgeo/geocaching/connector/lc/LCConnector.java [hunk 1]
```diff
@@ -64,7 +64,7 @@ public boolean canHandle(@NonNull final String geocode) {
     @NonNull
     public String getCacheUrl(@NonNull final Geocache cache) {
         final String launcher = Settings.getALCLauncher();
-        if (launcher.equals(null) || launcher.equals("")) {
+        if (launcher == null || "".equals(launcher)) {
```
[the remark this answers is not in the document]
Comment (baiti) [reply main/src/cgeo/geocaching/connector/lc/LCConnector.java:67]: @fm-sys ... much better solution. Will do once the dust around the git magic settles.

**Comment by moving-bits** [comment]
> Perhaps master is anyway the right branch for this function.

It would be nice to have it in the next release already, to start with a complete LC implementation, so I would propose to target it to release.

**Comment by baiti** [comment]
> > Perhaps master is anyway the right branch for this function.
> 
> It would be nice to have it in the next release already, to start with a complete LC implementation, so I would propose to target it to release.

Yes, you are right. And with @eddiemuc's magic commands above. I have actually go everything in sync again and created a new PR based on release and it looks good. We can later close and ignore this one here once the other one has been merged I'd say.

BTW: I  would assume you all know how to play with the capabilities that lie within this PR?

**Comment by Lineflyer** [comment]
Obsolete.

**Pull request closed without merging**